client/templates/dashboard/dashboard.html

```diff
@@ -2,10 +2,105 @@
   {{> afNavbar}}
   <div class='container'>
     <div class='row'>
-      <a class='btn btn-default' href="{{pathFor 'allShipping'}}">View All Orders Waiting to Be Shipped</a>
+      <h3>Printing Orders</h3>
+      <br>
+      <div class='row'>
+        <div class='col-xs-10'>
+          <a class='btn btn-lg btn-block btn-info print-specific-date' data-todays-date='{{chosenDate}}' href="{{pathFor 'orders.printAllForDate' date=chosenDate }}">Print {{chosenDateText}}&#39;s Shipping Invoices</a>
+        </div>
+        <div class='col-xs-2'>
+          <div class='picker'>
+            <input type="text" class="form-control input-lg" placeholder='select date' value='{{chosenDate}}' id='print-date'>
+          </div>
+        </div>
+      </div>
     </div>
+    <hr>
     <div class='row'>
-      <a class='btn btn-default' href="{{pathFor 'dateShipping' date=dateToday}}">All Orders that need to be shipped today</a>
+      <div class="col-xs-12 col-sm-6">
+        <a class='btn btn-lg btn-default' href="{{pathFor 'allShipping'}}">View All Orders Waiting to Be Shipped</a>
+      </div>
+      <div class="col-xs-12 col-sm-6">
+        <a class='btn btn-lg btn-default' href="{{pathFor 'dateShipping' date=dateToday}}">All Orders that need to be shipped today</a>
+      </div>
+    </div>
+    <div class='row'>
+
+    </div>
+    <div class='row'>
+      <h3>Orders Shipping in Near Future</h3>
+      <table class="table table-hover">
+        <thead>
+          <tr>
+            <th>Time</th>
+            <th>Created</th>
+            <th>Printed</th>
+            <th>Picking</th>
+            <th>Picked</th>
+            <th>Packing</th>
+            <th>Packed</th>
+            <th>Ready to Ship</th>
+            <th>Shipped</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Yesterday</td>
+            <td>{{yesterdaysOrders.orderCreated}}</td>
```

Comment:
We can dry this up a bit by looping through the relevant statuses and days that we are interested in.